### DISCUSSION TITLE
replace image of layer  in psd

### DISCUSSION THREAD

**Discussion by code2roc** [dlemstra/Magick.NET#1147] [Help]
hello
l first use this class library  and this is so good
l use the MagickImageCollection object to load psd file to get the layer collection successfully
l want to replace the image of one layer but l do not know how to use code to realize
please give me some advice, thank you so much!

**Comment by dlemstra** [comment]
One of the images in the `MagickImageCollection` is probably one of the images you are searching for. You will need to replace that image in the list with your image and `Write` the file again.
